### PR TITLE
@dzucconi: mixpanel -> segment (mixpanel + marketo)

### PR DESCRIPTION
**Pull request by dylanfareed**
updates dev to ruby 2.3.0
use middleman dotenv
set _mkt_trk cookie value in with the salesforce form
uses adjust link to itunes preview rather than direct itunes URL

### CODE REVIEW

File: source/javascripts/app.coffee [hunk 1]
```diff
@@ -10,6 +11,9 @@ window.App = class App
     modal = new Modal 'Thanks!<br>Your application has been submitted.'
     modal.open()
 
+  setMktoTrkVal: ->
+    $("input[name='00NC0000006VbjC']").val Cookies.get('_mkto_trk')
```
Comment (dzucconi) [inline source/javascripts/app.coffee]: I don't think `Cookies` is actually defined here?

**Comment by dzucconi** [comment]
Thanks—this looks good. Where's that `Cookies` coming from though?

File: source/layouts/layout.haml [hunk 2]
```diff
@@ -10,8 +10,8 @@
     %title= current_page.data.title || 'Folio: Artsy for Galleries'
 
     = stylesheet_link_tag 'all'
-    = javascript_include_tag ['//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js', '//fast.fonts.net/jsapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.js', 'all']
-    = partial 'mixpanel.html'
+    = javascript_include_tag ['//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js', '//fast.fonts.net/jsapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.js', '//cdnjs.cloudflare.com/ajax/libs/js-cookie/2.1.0/js.cookie.min.js', 'all']
```
Comment (dylanfareed) [inline source/layouts/layout.haml]: Cookies comes from the sneaky addition of `js-cookie`

Comment (dzucconi) [inline source/layouts/layout.haml]: :cookie:

**Comment by dzucconi** [comment]
Thankks

**Pull request merged**